Failing unit test

**Describe the bug**

I noticed this pattern in `pip/tests/test_interpreter.py`:
```python
195|     assert str(excinfo.value).startswith("Qsc.Resolve.NotFound") != 1
```

But that will always pass because `str.startswith(...)` returns a `bool` and in Python:
```python
False != -1
> True

True != -1
> True
```

When we remove the `!= -1` the test `test_target_error` is failing with error:
```console
E       AssertionError: assert False
E        +  where False = <built-in method startswith of str object at 0x0000027DFAFE1EF0>('Qsc.BaseProfCk.ResultLiteral')
E        +    where <built-in method startswith of str object at 0x0000027DFAFE1EF0> = 'Qsc.CapabilitiesCk.UseOfDynamicBool (https://aka.ms/qdk.qir#use-of-dynamic-bool)\n\n  × cannot use a dynamic bool val...eturn within a scope that depends on a measurement result\n        is not supported by the configured target profile\n'.startswith
E        +      where 'Qsc.CapabilitiesCk.UseOfDynamicBool (https://aka.ms/qdk.qir#use-of-dynamic-bool)\n\n  × cannot use a dynamic bool val...eturn within a scope that depends on a measurement result\n        is not supported by the configured target profile\n' = str(QSharpError('Qsc.CapabilitiesCk.UseOfDynamicBool (https://aka.ms/qdk.qir#use-of-dynamic-bool)\n\n  × cannot use a dyna...turn within a scope that depends on a measurement result\n        is not supported by the configured target profile\n'))
E        +        where QSharpError('Qsc.CapabilitiesCk.UseOfDynamicBool (https://aka.ms/qdk.qir#use-of-dynamic-bool)\n\n  × cannot use a dyna...turn within a scope that depends on a measurement result\n        is not supported by the configured target profile\n') = <ExceptionInfo QSharpError('Qsc.CapabilitiesCk.UseOfDynamicBool (https://aka.ms/qdk.qir#use-of-dynamic-bool)\n\n  × ca...in a scope that depends on a measurement result\n        is not supported by the configured target profile\n') tblen=1>.value
```

--- a/pip/tests/test_interpreter.py
+++ b/pip/tests/test_interpreter.py
@@ -184,7 +184,7 @@ def test_target_error() -> None:
         e.interpret(
             "operation Program() : Result { use q = Qubit(); if M(q) == Zero { return Zero } else { return One } }"
         )
-    assert str(excinfo.value).startswith("Qsc.BaseProfCk.ResultLiteral") != -1
+    assert str(excinfo.value).startswith("Qsc.CapabilitiesCk.UseOfDynamicBool")
 
 
 def test_qirgen_compile_error() -> None:
@@ -192,7 +192,7 @@ def test_qirgen_compile_error() -> None:
     e.interpret("operation Program() : Int { return 0 }")
     with pytest.raises(QSharpError) as excinfo:
         e.qir("Foo()")
-    assert str(excinfo.value).startswith("Qsc.Resolve.NotFound") != -1
+    assert str(excinfo.value).startswith("Qsc.Resolve.NotFound")
 
 
 def test_error_spans_from_multiple_lines() -> None:

